fastlane/lib/assets/Actions.md.erb

```diff
@@ -30,6 +30,8 @@ import './path/to/other/Fastfile'
 
 <%= action.description %>
 
+<%= "> #{action.deprecated_notes.gsub("\n\n", "\n")}" unless action.deprecated_notes.to_s.empty? %>
```

Comment:
Could we even highlight this to be red? Maybe using the super fancy colored boxes?